### PR TITLE
Eliminate creation and maintenance of unused assets #68

### PR DESCRIPTION
**Pull request by markus-ntt-seidl**
Eliminate creation and maintenance of unused assets #68

### CODE REVIEW

**Review by dubrie — APPROVED** [review]

**Comment by dubrie** [comment]
Initial review looks good, assigning to @dubrie for SME Review

File: docs/catalog/cloud/remove-unused-assets.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Inspect storage buckets if unused assets can be removed.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+With respect to the SCI equation, remove unused assets will impact one part:
+
+- `E`: Reduces the embodied emissions by not using the resources
```
Comment (dubrie) [inline docs/catalog/cloud/remove-unused-assets.md:30]: Should we include `M` here to account for the saved embodied carbon in storage resources after removing the assets?

File: docs/catalog/cloud/remove-unused-assets.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Assumptions
+
+- Assumes that resources can be decommissioned
+
+## Considerations
+
+- Before decommissioning resources consider workloads that run rarely, for example generating quarterly or yearly reports
```
Comment (dubrie) [inline docs/catalog/cloud/remove-unused-assets.md:38]: I'm not sure what this means or is referring to?

**Review by dubrie — CHANGES_REQUESTED** [review]
@markus-ntt-seidl A few small suggestions here but otherwise looks good.

File: docs/catalog/cloud/remove-unused-assets.md (hunk 2 above)
Comment (markus-ntt-seidl) [reply docs/catalog/cloud/remove-unused-assets.md:38]: This pattern is about removing unused assets. Unused is determined by the user of the cloud account and is not done by a computer. I assume that one would look at a log and see, this haven't been used for 2 weeks -> can be shut down. That would be harmful for, maybe, monthly database cleanup jobs which runs every 4weeks. (just an example).

This is following the rule, that every pattern must have an assumption / consideration to pass SME review.

File: docs/catalog/cloud/remove-unused-assets.md (hunk 1 above)
Comment (markus-ntt-seidl) [reply docs/catalog/cloud/remove-unused-assets.md:30]: Just noticing that E is wrong :(

File: docs/catalog/cloud/remove-unused-assets.md (hunk 2 above)
Comment (dubrie) [reply docs/catalog/cloud/remove-unused-assets.md:39]: "This is following the rule, that every pattern must have an assumption / consideration to pass SME review" -- I think that is a general guideline, not a hardened rule. 

That said, maybe it's a little clearer to say you can consider prioritizing decommissioning resources that that infrequently run? I think the wording here doesn't have an action with it other than to consider which makes it unclear what the user is supposed to be considering about it.

**Review by ccasher — APPROVED** [review]
Looks good to me!

**Review by camcash17 — APPROVED** [review]

**Pull request merged**